lib/dradis/plugins/import/result.rb

```diff
@@ -1,8 +1,9 @@
 module Dradis::Plugins::Import
   class Result
-    attr_accessor :description, :id, :tags, :title
+    attr_accessor :callback, :description, :id, :tags, :title
 
     def initialize(args={})
+      @callback    = args[:callback]    || Proc.new {}
```

Comment:
AFAIK we're not following this syntax anymore, can we update the initialize method accordingly?
```suggestion
      @callback = args[:callback] || Proc.new {}
```